primitives/Cargo.toml

```diff
@@ -26,6 +26,7 @@ hex = ["dep:hex", "hashes/hex", "internals/hex"]
 hashes = { package = "bitcoin_hashes", path = "../hashes", default-features = false }
 internals = { package = "bitcoin-internals", path = "../internals" }
 units = { package = "bitcoin-units", path = "../units", default-features = false }
+arrayvec = { version = "0.7", default-features = false }
```

Comment:
In e3b059cebfb917c3a876cc5a39d942d2dead475c:

This may interact with #4330 -- in a previous version of this PR you had 0.7.6 here. hex-conservative has 0.7.2. Our minimal lockfile has 0.7.4 for some reason.

Will ack and merge this regardless.